app/views/job_templates/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= include_javascript %>
+<%= include_javascript if SETTINGS[:version].short == '1.9' %>
```

Comment:
> Being compatible with latest stable has also it's advantages: you can notice that we don't hit CI errors in the development process

But we're also not running tests against develop so we're just getting this benefit at the expense of not having insight into changes in Foreman develop.  We're also about to branch for 1.10, so it seems likely we'll GA REX against 1.10.

Testing against 1.9 and develop when I looked into it wasn't something Jenkins could easily be changed to do, as it makes some assumptions about plugins.

Why do we need to do things so differently than other plugins do?

> I've seen many times this causing issues in Katello when running against nightlies.

Katello is a special case, but what's the problem with this?

The particular cases when Foreman changes break Katello nightly is important feedback, because we develop against `develop` - so we're ready for the next release.  Generally we're aware of these before they're even committed to Foreman because of tests are done against Katello in Foreman PR's.

The cases where we're not aware well in advance is because of lack of test coverage in Katello on Foreman integration, etc, and isn't an argument to develop against stable.